server/appengine/src/main/java/who/WhoServletModule.java

```diff
@@ -33,8 +33,6 @@ protected void configureServlets() {
     serve("/remote_api").with(new RemoteApiServlet());
 
     serve("/app").with(new AppStoreServlet());
-    serve("/terms").with(forwardTo("/terms.pdf"));
```

Comment:
Q: Do you want these to redirect to the WHO.int docs so that the links in the existing apps deployed go to the right place?

Reply:
WHO still haven't published those pages yet. Need to push on that through #1510. I realize this is going to break the existing links in the app... but then it's referring to old documents anyway. I'll separate out the server removal and merge that after the client is switched over. At that point, we'll publish a new client and perhaps test disabling the old ones.